team/bundles/org.eclipse.core.pki/plugin.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<plugin>
+	<extension point="org.eclipse.ui.preferencePages">
+		<page
+			category="org.eclipse.ui.net.NetPreferences"
+            class="org.eclipse.pki.preferences.PreferencePage"
+            id="org.eclipse.preferences.pkcs"
+            name="Java KeyStore Setup">
```

Comment:
missed i18n

Reply:
Where does that go?   on the name portion?

Reply:
Tried to find another plugin.xml file so I could see where the i18n is supposed to be...  None that I looked at had it??